Cesium.CodeGen/Ir/Expressions/BinaryOperatorExpression.cs

```diff
@@ -25,7 +26,22 @@ public BinaryOperatorExpression(Ast.BinaryOperatorExpression expression)
         Right = right.ToIntermediate();
     }
 
-    public virtual IExpression Lower() => new BinaryOperatorExpression(Left.Lower(), Operator, Right.Lower());
+    public virtual IExpression Lower() => Operator switch
+    {
+        BinaryOperator.GreaterThanOrEqualTo => new BinaryOperatorExpression(
+            new BinaryOperatorExpression(Left.Lower(), BinaryOperator.LessThan, Right.Lower()),
+            BinaryOperator.EqualTo,
+            new ConstantExpression(new IntegerConstant("0"))),
+        BinaryOperator.LessThanOrEqualTo => new BinaryOperatorExpression(
+            new BinaryOperatorExpression(Left.Lower(), BinaryOperator.GreaterThan, Right.Lower()),
+            BinaryOperator.EqualTo,
+            new ConstantExpression(new IntegerConstant("0"))),
+        BinaryOperator.NotEqualTo => new BinaryOperatorExpression(
+            new BinaryOperatorExpression(Left.Lower(), BinaryOperator.EqualTo, Right.Lower()),
+            BinaryOperator.EqualTo,
+            new ConstantExpression(new IntegerConstant("0"))),
```

Comment:
All these definitions will break for `NaN` when we support floats 😈 

For now, it's fine I guess, but let's open a new issue to not forget to handle it properly.

Reply:
Done in #117.